docs/using-the-compiler.rst

```diff
@@ -44,6 +44,48 @@ If ``solc`` is called with the option ``--link``, all input files are interprete
 
 If ``solc`` is called with the option ``--standard-json``, it will expect a JSON input (as explained below) on the standard input, and return a JSON output on the standard output.
 
+.. _compiler-version:
+.. index:: ! compiler version, compile version
+
+Setting the compiler version
+******************************
+
+When you compile your contract code you can specify the compiler version to target or avoid particular features or behaviours in the release.
+
+.. warning::
+
+   Compiling for the wrong EVM version can result in wrong, strange and failing
+   behaviour. Please ensure, especially if running a private chain, that you
+   use the correct EVM version.
+
+You use the ``--evm-version`` option on the command line:
+
+.. code-block:: shell
+
+  solc   --evm-version <VERSION> contract.sol
+
+Or if using :ref:`JSON description format <compiler-api>`, with the ``evmVersion`` key:
```

Comment:
We usually call this `standard-json-io` (but please look up the correct term in the documentation)

Reply:
@chriseth I couldn't find that mentioned anywhere, but 'JSON input' seemed to be the only consistent term I could fin.